src/script/lua_api/l_localplayer.cpp

```diff
@@ -176,6 +176,9 @@ int LuaLocalPlayer::l_get_physics_override(lua_State *L)
 	lua_pushboolean(L, player->physics_override_new_move);
 	lua_setfield(L, -2, "new_move");
 
+	lua_pushboolean(L, player->physics_override_new_move);
```

Comment:
copy paste mistake :)
```suggestion
	lua_pushboolean(L, player->physics_override_new_jump);
```